### PR TITLE
Minor updates to the Getting Started with Security doc

### PR DESCRIPTION
**Pull request by sberyozkin**
This PR:

* Removes the bullets from the section describing how to test the application in devmode. Rolfe proposed to remove one bullet, but the remaining ones there seem to impact it as well
* Suggests an alignment fix of the independent/sibling section on how to test with curl or the browser

### CODE REVIEW

File: docs/src/main/asciidoc/security-getting-started-tutorial.adoc [hunk 1]
```diff
@@ -503,15 +503,19 @@ While developing your application, you can add and run tests individually by usi
 Dev Services for PostgreSQL supports testing while you develop by providing a separate PostgreSQL test container that does not conflict with the dev mode container.
 ====
 
-=== Use curl or a browser to test your application
+== Test your application using Curl or browser
+
+To test your application using Curl or the browser, you must first start a PostgreSQL server, and compile and run your application either in JVM or native mode.
```
Comment (rolfedh) [inline docs/src/main/asciidoc/security-getting-started-tutorial.adoc:508]:
```suggestion
To test your application using Curl or the browser, you must first start a PostgreSQL server, then compile and run your application either in JVM or native mode.
```
Or
```suggestion
To test your application with Curl or a browser, first start a PostgreSQL server, then compile and run the application in JVM or native mode.
```

**Review by rolfedh — APPROVED** [review]
LGTM. Excellent improvements! I've offered a very minor suggestion.

**Comment by sberyozkin** [comment]
Thanks @rolfedh

**Comment by sberyozkin** [comment]
It should be ready to go now that Rolfe approved it, hopefully it can make it to 3.15

**Review by gastaldi — APPROVED** [review]

**Pull request merged**